c_api/concurrentqueue.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifdef _WIN32
+#if defined(MOODYCAMEL_STATIC) //preferred way
+#define MOODYCAMEL_EXPORT
+#elif defined(DLL_EXPORT)
+#define MOODYCAMEL_EXPORT __declspec(dllexport)
+#else
+#define MOODYCAMEL_EXPORT __declspec(dllimport)
+#endif
+#else
+	
+#error *nix support is untested
+/*
+need non-windows version here
+*/
+#endif
+
+#define MOODYCAMEL_OK 0
+#define MOODYCAMEL_ERR -1
+#define MOODYCAMEL_BAD_HANDLE -2
+#define MOODYCAMEL_BAD_VALUE_HANDLE -3
+#define MOODYCAMEL_VALUE_NOT_FOUND 1
```

Comment:
Why is this the only positive error?

Reply:
I was thinking it was only a "warning" type of error.